pkg/apis/admissionregistration/types.go

```diff
@@ -118,35 +118,65 @@ const (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ExternalAdmissionHookConfiguration describes the configuration of initializers.
-type ExternalAdmissionHookConfiguration struct {
+// ValidatingWebhookConfiguration describes the configuration of and admission webhook that accept or reject and object without changing it.
+type ValidatingWebhookConfiguration struct {
 	metav1.TypeMeta
 	// Standard object metadata; More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata.
 	// +optional
 	metav1.ObjectMeta
-	// ExternalAdmissionHooks is a list of external admission webhooks and the
-	// affected resources and operations.
+	// Webhooks is a list of webhooks and the affected resources and operations.
 	// +optional
-	ExternalAdmissionHooks []ExternalAdmissionHook
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	Webhooks []Webhook
```

Comment:
Webhook or AdmissionHook? I have a very very mild preference for the latter.

Reply:
We remove admission from the type name because it was the duplicate of (part of) the package name. I would argue the same here that I prefer Webhook for that reason.

Reply:
+1 for Webhook